packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -277,6 +285,18 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
     return '';
   }
 
+  private _getAriaValueNow(component: Exclude<ColorComponent, 'hex'>): number {
+    const { color, editingColor } = this.state;
+
+    // Test for editingColor.value !== '' to ensure that we don't update aria-valuenow if input is empty string
+    if (editingColor && editingColor.component === component && editingColor.value !== '') {
+      const maxValue = component === 'a' || component === 't' ? MAX_COLOR_ALPHA : MAX_COLOR_RGB;
+      return clamp(Number(editingColor.value), maxValue);
```

Comment:
This might work but I also wonder if the mismatch between the typed value and the announced value could cause issues: in this case if they type 300, that value will remain in the field until they focus out, but the announced value will be 255.

Reply:
Yeah it wasn't great, didn't love the fix. 
An Accessibility expert came back on the issue and suggested using a live region (aria-live). It works like a charm, I switched to that change instead. Waaaay better.